Library/ViewModels/EditorialProjectsViewModel.swift

```diff
@@ -51,7 +56,21 @@ public class EditorialProjectsViewModel: EditorialProjectsViewModelType,
 
   public let configureDiscoveryPageViewControllerWithParams: Signal<DiscoveryParams, Never>
   public let dismiss: Signal<(), Never>
+  public let imageName: Signal<String, Never>
+  public let titleLabelText: Signal<String, Never>
 
   public var inputs: EditorialProjectsViewModelInputs { return self }
   public var outputs: EditorialProjectsViewModelOutputs { return self }
 }
+
+private func editorialImageName(for tagId: DiscoveryParams.TagID) -> String {
+  switch tagId {
+  case .goRewardless: return "go-rewardless-home"
+  }
+}
+
+private func editorialTitleLabelText(for tagId: DiscoveryParams.TagID) -> String {
+  switch tagId {
+  case .goRewardless: return Strings.This_holiday_season_support_a_project_for_no_reward()
```

Comment:
This _may_ be the wrong string: https://github.com/kickstarter/android-oss/pull/683#discussion_r349847955